dev/ansible/03_run_tests.yml

```
@@ -1,6 +1,6 @@
 - name: set variable
   set_fact:
-    consul_token: "master"
+    consul_token: "{{ lookup('env', 'consul_master_token') }}"
```

I would recommend staying consistent.
```suggestion
    consul_token: "{{ lookup('env', 'consul_master_token') }}"
    vault_addr: "{{ lookup('env', 'VAULT_ADDR') }}"
    vault_token: "{{ lookup('env', 'vault_master_token') }}"
```